src/Apps/Order/Components/PaymentPicker.tsx

```diff
@@ -404,6 +424,10 @@ export const PaymentPickerFragmentContainer = createFragmentContainer(
           state
           country
           postal_code
+          expiration_month
+          expiration_year
+          last_digits
+          brand
```

Comment:
Unrelated to this PR, A nice follow-up PR might be to move the radio buttons inside a fragment container so that the main component doesn't need to care about these fields.